sky/clouds/local.py

```diff
@@ -115,6 +115,17 @@ def get_feasible_launchable_resources(self,
         )
         return [resources], []
 
+    def accelerator_in_region_or_zone(self,
+                                      accelerator: str,
+                                      acc_count: int,
+                                      region: Optional[str] = None,
+                                      zone: Optional[str] = None) -> bool:
+        return True
+
+    @classmethod
+    def get_zone_shell_cmd(cls) -> Optional[str]:
+        return None
```

Comment:
Consider: moving this to parent class, Cloud? Otherwise, say a new Cloud is implemented, it forgot to implement this method then SkyPilot will error.

Reply:
The original method in `Cloud` is this:
```
    @classmethod
    def get_zone_shell_cmd(cls) -> Optional[str]:
        """Returns the shell command to obtain the zone of instance."""
        raise NotImplementedError
```
It will error, similar to the second case mentioned above.